--- samples/bnfcheck.cpp 2017-04-05 18:34:19.979855428 +0100
+++ samples/bnfcheck.cpp 2017-04-05 18:36:50.976991804 +0100
@@ -19,6 +19,7 @@
 
 #include <string>
 #include <iostream>
+#include <stdio.h>
 
 #include <SimpleOpt.h>
 
@@ -121,7 +122,7 @@
 "  -v LEVEL, --verbose=LEVEL set verbosity to LEVEL (default %i)\n"
 "  --help                    display this help and exit\n"
 "\n"
-"Report bugs to <"PACKAGE_BUGREPORT">.\n",
+"Report bugs to <" PACKAGE_BUGREPORT ">.\n",
           argv[0], delimiter, test.get_verbose_level()
         );
         exit(0);
